helm/ako/crds/ako.vmware.com_aviinfrasettings.yaml

```diff
@@ -100,6 +100,8 @@ spec:
                     - LARGE
                     - DEDICATED
                     type: string
+                  dedicatedGatewayMode:
```

Comment:
Can we decide if we really need a field in AviInfraSetting when the decision is solely based on the hostname field in Gateway and HTTPRoute spec. Can we not always consider the case of empty and/or "*" hostname in (gateway+httproute) to be dedicatedGatewayMode?